### PR TITLE
Playing cards added

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#95 

new 1-line arts added:
+ playing cards waterfall
+ playing cards waterfall (trump)
+ playing cards hearts waterfall
+ playing cards diamonds waterfall
+ playing cards clubs waterfall
+ playing cards spades waterfall
+ joker1
+ joker2
+ joker3
+ joker4

### CODE REVIEW

File: art/art_dic.py [hunk 1]
```diff
@@ -566,10 +566,17 @@
...
+    "playing cards waterfall (trump)": "🃠🃡🃢🃣🃤🃥🃦🃧🃨🃩🃪🃫🃬🃭🃮🃯🃰🃱🃲🃳🃴🃵",
     "playing cards hearts": "[♥]]]",
+    "playing cards hearts waterfall": "🂠🂱🂲🂳🂴🂵🂶🂷🂸🂹🂺🂻🂼🂽🂾",
     "playing cards diamonds": "[♦]]]",
-    "playing cards spades": "[♠]]]",
+    "playing cards diamonds waterfall": "🃟🃁🃂🃃🃄🃅🃆🃇🃈🃉🃊🃋🃌🃍🃎",
     "playing cards clubs": "[♣]]]",
+    "playing cards clubs waterfall": "🃏🃑🃒🃓🃔🃕🃖🃗🃘🃙🃚🃛🃜🃝🃞",
+    "playing cards spades": "[♠]]]",
+    "playing cards spades waterfall": "🃠" + "🂡🂢🂣🂤🂥🂦🂧🂨🂩🂪🂫🂬🂭🂮",
```
Comment (sadrasabouri) [inline art/art_dic.py:579]: Here, two string are concatenated by `+` because of a weird manner.

File: art/art_dic.py [hunk 2]
```diff
@@ -566,10 +566,17 @@
     "boombox2": "♫♪ |̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅| ♫♪",
     "equalizer": "▇ ▅ █ ▅ ▇ ▂ ▃ ▁ ▁ ▅ ▃ ▅ ▅ ▄ ▅ ▇",
     "playing cards": "[♥]]] [♦]]] [♣]]] [♠]]]",
+    "playing cards waterfall": "🂱🂲🂳🂴🂵🂶🂷🂸🂹🂺🂻🂼🂽🂾🃁🃂🃃🃄🃅🃆🃇🃈🃉🃊🃋🃌🃍🃎"
+                               "🃑🃒🃓🃔🃕🃖🃗🃘🃙🃚🃛🃜🃝🃞🂡🂢🂣🂤🂥🂦🂧🂨🂩🂪🂫🂬🂭🂮🂠🃏🃟",
+    "playing cards waterfall (trump)": "🃠🃡🃢🃣🃤🃥🃦🃧🃨🃩🃪🃫🃬🃭🃮🃯🃰🃱🃲🃳🃴🃵",
     "playing cards hearts": "[♥]]]",
+    "playing cards hearts waterfall": "🂠🂱🂲🂳🂴🂵🂶🂷🂸🂹🂺🂻🂼🂽🂾",
```
Comment (sadrasabouri) [inline art/art_dic.py:573]: These set waterfalls became well-ordered so that indexing will be easy for user.

File: art/art_dic.py (hunk 1 above)
Comment (sadrasabouri) [reply art/art_dic.py]: It might be a special case in vscode, it was alright with vim and other terminal-base editors.

**Review by sepandhaghighi — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
LGTM 💯

**Pull request merged**